src/codegen/function_body_chunk.rs

```diff
@@ -1159,7 +1146,18 @@ impl Builder {
         mem_mode: &OutMemMode,
         uninitialized_vars: &mut Vec<(String, bool)>,
     ) -> Chunk {
-        let value = Chunk::Custom(parameter.name.clone());
+        let value = if let Some(need_from_glib) =
+            self.is_uninitialized_var(&parameter.name, uninitialized_vars)
+        {
+            Chunk::Custom(format!(
```

Comment:
This seems to fail for bool out parameters

```
 error[E0277]: the trait bound `bool: glib::translate::FromGlib<MaybeUninit<i32>>` is not satisfied
Error:     --> gtk/src/auto/icon_info.rs:152:72
     |
152  |             let result = if error.is_null() { Ok((from_glib_full(ret), from_glib(was_symbolic))) } else { Err(from_glib_full(error)) };
     |                                                                        ^^^^^^^^^ the trait `glib::translate::FromGlib<MaybeUninit<i32>>` is not implemented for `bool`
     |
```

Reply:
Issue was in async trampoline.